zerver/actions/streams.py

```diff
@@ -787,9 +788,11 @@ def bulk_add_subscriptions(
     altered_user_dict: dict[int, set[int]] = defaultdict(set)
     altered_guests: set[int] = set()
     altered_streams_dict: dict[UserProfile, set[int]] = defaultdict(set)
+    stream_to_subscribers: dict[Stream, set[UserProfile]] = defaultdict(set)
```

Comment:
`subscriber_count_changes` seems like a better name.

Reply:
I don't agree, but ok

Reply:
That doesn't at all communicate that it's a map of deltas?